### PR TITLE
Add email verification (OTP) to other policies

### PR DESCRIPTION
**Pull request by nomasprime**

### CODE REVIEW

**Review by WorkSutton — CHANGES_REQUESTED** [review]
I believe we want to be consistent and have a single set of code for OTP that live under the generic area of claim. Both for the views and translations as if I have understood the JIRA correct is become used across all 3 existing Policies.

File: app/views/maths_and_physics/claims/_one_time_password.html.erb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        <%= errors_tag current_claim, :one_time_password %>
+        <%= form.text_field :one_time_password,
+                            autocomplete: "off",
+                            class: css_classes_for_input(current_claim, :one_time_password, 'govuk-input--width-10'),
+                            "aria-describedby" => "one-time-password-hint" %>
+      <% end %>
+
+
+      <div class="govuk-!-margin-bottom-6">
+        <%= link_to "Not received an email", claim_path(EarlyCareerPayments.routing_name, "email-address"), class: "govuk-link govuk-link--no-visited-state " %>
```
[the remark this answers is not in the document]
Comment (nomasprime) [reply app/views/maths_and_physics/claims/_one_time_password.html.erb:28]: Well caught.

File: app/views/maths_and_physics/claims/email_verification.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<% content_for(:page_title, page_title(t("early_career_payments.one_time_password"), policy: current_policy_routing_name, show_error: current_claim.errors.any?)) %>
```
[the remark this answers is not in the document]
Comment (nomasprime) [reply app/views/maths_and_physics/claims/email_verification.html.erb:1]: Can't find claim section, meant top level?

**Review by WorkSutton — APPROVED** [review]
LGTM now with the changes to making a generic process with generic claim views/locale info.

**Pull request merged**